letsencrypt-apache/letsencrypt_apache/configurator.py

```diff
@@ -493,7 +493,10 @@ def prepare_server_https(self, port, temp=False):
         """
         if "ssl_module" not in self.parser.modules:
             logger.info("Loading mod_ssl into Apache Server")
-            self.enable_mod("ssl", temp)
+            if self.config.func.__name__ == "auth":
```

Comment:
`IConfig.func` is not part of the API and **this line is very likely to break**! Presence of `func` is a huge coincidence. Could you please suggest alternative implementations that do not exploit this?